cltk/tokenize/latin_exceptions.py

```diff
@@ -14,41 +14,106 @@
+ne_exceptions += ['absone', 'acharne', 'acrisione', 'acumine', 'adhucine', 'adsuetudine', 'aeetine', 'aeschynomene',
```

Comment:
Same issue here, but what characterizes the superset of vocabulary that these exceptions encompass?  Do tell.

Reply:
@BananaNosh please clarify the sources here too, and what parts of the Latin language they cover.

Reply:
See above.